helpers/github.ts

```diff
@@ -129,12 +130,24 @@ export function getDevpoolIssueLabels(issue: GitHubIssue, projectUrl: string) {
   // get owner and repo name from issue's URL because the repo name could be updated
   const [ownerName, repoName] = getRepoCredentials(issue.html_url);
 
+  const pricing = getIssuePriceLabel(issue)
+
+  let devpoolIssueLabels: string[];
+
   // default labels
-  const devpoolIssueLabels = [
-    getIssuePriceLabel(issue), // price
-    `Partner: ${ownerName}/${repoName}`, // partner
-    `id: ${issue.node_id}`, // id
-  ];
+  if (pricing != "Pricing: not set") {
```

Comment:
Perhaps this should be an environment variable instead of a string

Reply:
I wonder if its a better idea to do `Pricing: -1` the idea is that I believe the parsers we use expect a number. We can just check if its less than 0. 

What is the best choice here @rndquu?

Reply:
> I wonder if its a better idea to do `Pricing: -1` the idea is that I believe the parsers we use expect a number. We can just check if its less than 0.
> 
> What is the best choice here @rndquu?

The `Pricing: not set` is used purely to distinguish issues with and without price labels. The text itself is not used anywhere so it doesn't matter much whether it's `Pricing: -1`, `Pricing: not set`, etc...

@xno-miner Could you move the `Pricing: not set` string to a separate constant variable since it's used in multiple places ([one](https://github.com/xno-miner/devpool-directory/blob/8cdeed44ab0a42a694aca9d7641593e8dec66131/helpers/github.ts#L138), [two](https://github.com/xno-miner/devpool-directory/blob/8cdeed44ab0a42a694aca9d7641593e8dec66131/helpers/github.ts#L209))?

Otherwise the PR works fine.

Reply:
Sorry for the delay, I only just now have noticed this message. I pushed the changes.